src/template.ts

```diff
@@ -108,6 +108,7 @@ canvas.show {
   width: 100%;
   height: 100%;
   position: absolute;
+  border: none;
```

Comment:
`<button>` has padding by default that we probably want to remove: `padding: 0;`